pom.xml

```diff
@@ -1656,12 +1660,12 @@
       <id>hadoop-2.4</id>
       <properties>
         <hadoop.version>2.4.0</hadoop.version>
-        <protobuf.version>2.5.0</protobuf.version>
+	<protobuf.version>${protobuf.version}</protobuf.version>
```

Comment:
Just delete this line instead. No point in setting the variable to the value it already has. Same in other places.